src/badguy/badguy.cpp

```diff
@@ -1066,7 +1068,8 @@ BadGuy::after_editor_set()
 
   const std::string direction_str = m_start_dir == Direction::AUTO ? "left" : dir_to_string(m_start_dir);
   bool action_set = false;
-  for (std::string action_str : {"editor", "normal", "idle", "flying", "walking", "standing", "swim"})
+  const std::list<std::string> actions = {"editor", "normal", "idle", "flying", "walking", "standing", "swim"};
```

Comment:
List was not to be taken literally, I think you can do fine with just std::string[], no?